react/src/lib/components/DeckGLMap/components/DistanceScale.tsx

```diff
@@ -1,26 +1,52 @@
 import React from "react";
 
 interface scaleProps {
+    // Needed the zoom value to calculate width in units
     zoom: number;
+    // Scale increment value
+    incrementValue: number;
+    // width per unit value to calculate width in units
+    widthPerUnit: number;
 }
-const DistanceScale: React.FC<scaleProps> = ({ zoom }: scaleProps) => {
-    const [width] = React.useState(100);
-    const pixelPerUnit = width / Math.pow(2, zoom);
-    const scaleBarStyle = {
-        width: width,
+
+const roundOffvalues = function (num: number, step: number) {
```

Comment:
This name leads me to think that this function receives many values (maybe an array) and rounds them.
I think it should be called something like `roundToStep` since it tries to round a value up or down to a specified step.